crates/containerd-shim-wasm/src/sandbox/cli.rs

```diff
@@ -1,3 +1,69 @@
+//! Command line interface for the containerd shim.
+//!
+//! The CLI provides the interface between containerd and the Wasm runtime.
+//! It handles commands like start and delete from containerd's shim API.
+//!
+//! ## Usage
+//!
+//! The shim binary should be named `containerd-shim-<name>-v1` and installed in $PATH.
+//! containerd will call the shim with various commands.
+//!
+//! ## Configuration
+//!
+//! The shim can be configured using the [`Config`] struct:
+//!
+//! ```rust, no_run
+//! use containerd_shim_wasm::Config;
+//!
+//! let config = Config {
+//!     // Disable automatic logger setup
+//!     no_setup_logger: false,
+//!     // Set default log level
+//!     default_log_level: "info".to_string(),
+//!     // Disable child process reaping
+//!     no_reaper: false,
```

Comment:
Note: Setting `no_reaper` to true makes the shim stop working. We should probably create an issue in rust-extensions.